src/flask_app.py

```diff
@@ -27,7 +27,7 @@ def input_data():
 	except Exception ,e:
 		status = 'error'
 		results = 'data format not recoginzed'
-		return jsonify({'status': status, 'error':str(e),'results':data}), 400
+		return jsonify({'status': status, 'error':str(e),'results':'no data in the results'}), 400
```

Comment:
`results` cannot be different formats
* for success [...] 
* for error "..."

It should be an array all the time, if it's presented in response.